app/src/main/java/io/gnosis/safe/Tracker.kt

```diff
@@ -198,6 +206,7 @@ class Tracker internal constructor(
         val KEY_GENERATED = "user_key_generated"
         val KEY_IMPORTED = "user_key_imported"
         val KEY_IMPORTED_LEDGER = "user_ledger_nano_x_key_imported"
+        val KEY_IMPORTED_KEYSTONE = "user_keystone_key_imported"
         val KEY_DELETED = "user_key_deleted"
         val TRANSACTION_CONFIRMED = "user_transaction_confirmed"
         val TRANSACTION_CONFIRMED_LEDGER = "user_transaction_confirmed_ledger_nano_x"
```

Comment:
please add 
```kotlin 
val TRANSACTION_CONFIRMED_KEYSTONE = "user_transaction_confirmed_keystone"
```
and
```kotlin 
val TRANSACTION_REJECTED_KEYSTONE = "user_transaction_rejected_keystone"
```
no need to wire these events for now. We will do it later ourselves.